webui.sh

```diff
@@ -212,5 +212,5 @@ else
     printf "Launching launch.py..."
     printf "\n%s\n" "${delimiter}"
     prepare_tcmalloc
-    exec "${python_cmd}" "${LAUNCH_SCRIPT}" "$@"
+    exec "${venv_dir}/bin/python3" "${LAUNCH_SCRIPT}" "$@"
```

Comment:
This change would make `python_cmd` useless for everyone, which is probably not what anyone wants.

Reply:
Starting the web-ui will create a virtual environment on which `source "${venv_dir}"/bin/activate` has been executed. His purpose is to use this Python interpreter. My modification is only for the Make the behavior more accurate on MacOS

https://github.com/AUTOMATIC1111/stable-diffusion-webui/blob/89f9faa63388756314e8a1d96cf86bf5e0663045/webui.sh#L168-L170